src/app/loaders/express/index.ts

```diff
@@ -159,6 +160,8 @@ const loadExpressApp = async (connection: Connection) => {
   // Registered routes with the Singpass/Corppass servers
   app.use('/singpass/login', SingpassLoginRouter)
   app.use('/corppass/login', CorppassLoginRouter)
+  // Registered routes with sgID
+  app.use('/sgid', SgidRouter)
```

Comment:
the singpass and corppass routes are stuck as they are due to a hardcoded NDI whitelist.

i don't think this is the case for `/sgid` right? Should we move it to a more semantically correct API endpoint `/api/v3/forms/:formId/auth/sgid` instead? 

the new router can reside in `PublicFormAuthRouter` :P

Reply:
The route was decided by @liangyuanruo . I have already raised the issue of placing the endpoint at the appropriate path.